kube-core/src/resource.rs

```diff
@@ -170,6 +174,10 @@ pub trait ResourceExt: Resource {
     fn finalizers(&self) -> &[String];
     /// Provides mutable access to the finalizers
     fn finalizers_mut(&mut self) -> &mut Vec<String>;
+    /// Returns the creation timestamp
+    ///
+    /// This is guaranteed to exist on resources received by the apiserver.
+    fn creation(&self) -> Option<Time>;
```

Comment:
`creation` sound awkward to me. `created_at`?

Reply:
hm, actually, it might be smarter to go for the full `creation_timestamp` name here upon reflection; because everything else is named exactly after the names in `ObjectMeta`

Reply:
Yeah, looks good to me